### PR TITLE
TMAI August,September,October,November MindBenders Club TCET

### PR DESCRIPTION
**Pull request by AI-ApeX-DeV**

### CODE REVIEW

**Review by demonkillerr — APPROVED** [review]
LGTM! Should work on image optimization.

**Comment by vish-rt** [comment]
Changing base to `blogs` as all blog PRs are meant to be made to that branch

File: blog/2023-10-17-tmai-september-23/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
```
Comment (vish-rt) [inline blog/2023-10-17-tmai-september-23/index.md:1]: Few of the metadata in this frontmatter are not necessary.
Read the [first steps](https://opensource.tcetmumbai.in/blog/blog-posting-guidelines#first-steps) of the posting guidelines for which metadata information should be used.

**Review by vish-rt — CHANGES_REQUESTED** [review]
Requesting some minor changes as follows:
1. Utilize [truncate](https://opensource.tcetmumbai.in/blog/blog-posting-guidelines#utilizing-truncate) in all blogs. It reduces the clutter on the blogs page itself and ensures smooth flow. Below is an example of how to truncate your blogs.
image
2. There's alot of metadata in the frontmatter of each blog, some are not working as intended.
I have added the review comments accordingly on what can be fixed to match with the site theme on the September blog. I recommend adding those fixes in all three blogs.

**Pull request closed without merging**